compiler/test

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 
-export COMPILER_ROOT=$PWD
+# Fix when run from a differen directory
+wd=`dirname $0`
+cd $wd
+
+export COMPILER_ROOT=`cd $wd; pwd`
```

Comment:
Do we need the cd $wd in line 7? It looks redundant with line 5.

Reply:
Not needed, this is fixed!